NSS-Day-Cohort-69/tabloid-tabloid-team-2-gonzos
# Ft/addcategory/zavier

### Purpose
To allow the admin to view all categories.

### Files Changed
- created controller for categories in `CategoryController.cs`
- created manager in `categoryManager.js`
- added Migrations/ to `.gitignore`
- deleted certain Migrations files
- added NavBar button for categories in `NavBar.js` and `ApplicationViews.js`
- created logic for displaying categories in `CategoryList.jsx`

### To Test
Given the program is running and the client side had be launched, login to Tabloid as an admin. Confirm there is a link on the navigation menu labeled 'Category'. Confirm clicking this link takes you to a new page where a list of categories is displayed.
